Vagrantfile

```diff
@@ -116,7 +117,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
               consul_join: consul_join,
               consul_retry_join: consul_retry_join,
               marathon_master_peers: marathon_master_peers,
-              marathon_zk_peers: marathon_zk_peers
+              marathon_zk_peers: marathon_zk_peers,
+              mesos_master_quorum: 2,
```

Comment:
Yes ... those two ... I prefer them to live in `vagrant.yml` to avoid hardcoding values here